atlasdb-autobatch/src/main/java/com/palantir/atlasdb/autobatch/Autobatchers.java

```diff
@@ -68,31 +68,24 @@ private Autobatchers() {}
         private final Function<Integer, EventHandler<BatchElement<I, O>>> handlerFactory;
 
         @Nullable private String purpose;
-        private int bufferSize = DEFAULT_BUFFER_SIZE;
 
         private AutobatcherBuilder(Function<Integer, EventHandler<BatchElement<I, O>>> handlerFactory) {
             this.handlerFactory = handlerFactory;
         }
 
-        public AutobatcherBuilder<I, O> safeLoggablePurpose(String purposeParam) {
+        public AutobatcherBuilder<I, O> safeLoggablePurpose(@CompileTimeConstant String purposeParam) {
```

Comment:
⭐️ (that said, someone might want to do `String.format(blah)`, but I doubt so)

Reply:
haha I hope not